<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_record_alarm</class>
 <widget class="QDialog" name="Dlg_record_alarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>报警回顾</string>
  </property>
  <widget class="QPushButton" name="m_btn_return">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>311</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd_fast">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back_end">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>|&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back_fast">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd_end">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;|</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_btn_style">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_record">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>记录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
